File: project/backend/models/comment.js
'use strict';
module.exports = (sequelize, DataTypes) => {
  const Comment = sequelize.define('Comment', {
    comment_to: {
      allowNull: false,
      type: DataTypes.STRING
    },
    rating: {
      allowNull: false,
      type: DataTypes.FLOAT
    },
    text: {
      allowNull: false,
      type: DataTypes.STRING
    },
    comment_by: {
      allowNull: false,
      type: DataTypes.STRING
    }
  });
  Comment.associate = function(models) {
    models.Comment.belongsTo(models.User, {
      foreignKey: 'comment_by',
      as: 'author',
      constraints: false
  });
  };
  return Comment;
};

File: project/mobile/bulingo/app/src/main/java/com/bulingo/Database/Comment.java
package com.bulingo.Database;

import com.google.gson.annotations.SerializedName;

public class Comment {

    @SerializedName("text")
    public String text;
    @SerializedName("rating")
    public String rating;
    @SerializedName("comment_by")
    public String comment_by;
    @SerializedName("comment_to")
    public String comment_to;
    @SerializedName("created_at")
    public String created_at;

}
